engine/pkg/cmd/cli/cli_job.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+	"time"
+
+	"github.com/pingcap/log"
+	"github.com/spf13/cobra"
+
+	"github.com/pingcap/tiflow/engine/client"
+	"github.com/pingcap/tiflow/engine/pkg/tenant"
+	"github.com/pingcap/tiflow/pkg/errors"
+)
+
+// jobGeneralOptions defines some general options of job management
+type jobGeneralOptions struct {
+	projectID string
+	tenantID  string
```

Comment:
It's redundant. The `ProjectInfo` already contains `tenantID` field.

Reply:
They are different, `tenantID` comes from command line parameter, `tenant` is generated from `projectID` and `tenantID`

Reply:
@CharlesCheung96 We need 'tenantID' and 'projectID' for Billing system in the future. They have different meaning actually.

In our implement, sometimes we only use 'projectID' for a unique identifier for multi-project isolation.